### ISSUE TITLE
Blank screen after restarting the CRM. found "INTERNAL_SERVER_ERROR",     "response": "Token has expired."  on network tab

### ISSUE THREAD

**Issue by Rounak826** [twentyhq/twenty#6648]
## Bug Description

A blank screen appears after successful login and restarting the CRM due to an expired token.

Network Tab
```{
    "errors": [
        {
            "message": "Error",
            "extensions": {
                "code": "INTERNAL_SERVER_ERROR",
                "response": "Token has expired."
            }
        }
    ]
}```
Server Logs
```Exception Captured
  { user: undefined }
  [
    AuthException [Error]: Token has expired.
        at TokenService.verifyJwt (/app/packages/twenty-server/dist/src/engine/core-modules/auth/services/token.service.js:367:23)
        at TokenService.validateToken (/app/packages/twenty-server/dist/src/engine/core-modules/auth/services/token.service.js:175:36)
        at GraphQLHydrateRequestFromTokenMiddleware.use (/app/packages/twenty-server/dist/src/engine/middlewares/graphql-hydrate-request-from-token.middleware.js:49:44)
        at /app/node_modules/@nestjs/core/router/router-proxy.js:9:23
        at Layer.handle [as handle_request] (/app/node_modules/@nestjs/platform-express/node_modules/express/lib/router/layer.js:95:5)
        at next (/app/node_modules/@nestjs/platform-express/node_modules/express/lib/router/route.js:144:13)
        at next (/app/node_modules/@nestjs/platform-express/node_modules/express/lib/router/route.js:140:7)
        at next (/app/node_modules/@nestjs/platform-express/node_modules/express/lib/router/route.js:140:7)
        at next (/app/node_modules/@nestjs/platform-express/node_modules/express/lib/router/route.js:140:7)
        at next (/app/node_modules/@nestjs/platform-express/node_modules/express/lib/router/route.js:140:7) {
      code: 'FORBIDDEN_EXCEPTION'
    }
  ]
```

## Expected behavior

Handle expired tokens gracefully and  prompt users to log in again or refresh the token internally

## Technical inputs

- I am using Self-Hosted Twenty CRM created with Docker Compose with the Latest tag.
- I am not sure about the version

**Comment by raphspeaks** [comment]
I’ve experienced the same thing. The stale token breaks the front end. When this happens you’ll see an error in the bottom right hand screen. I would expect that the application would route to a login screen, but instead it’s just blank.

**Comment by Rounak826** [comment]
I have found a temporary workaround to log into my account by navigating to the application section in the inspect window and manually deleting the token. but it's very frustrating. The worst part is my non-tech team can't access the CRM.

**Comment by Rounak826** [comment]
Yes, that's the only option for now. I hope this issue gets fixed soon

**Comment by agerman77** [comment]
You have to delete the tokenPair cookie manually. Any workaround for this?

![image](https://github.com/user-attachments/assets/53716da2-52b5-4678-a259-178af15b1d22)

**Comment by Rounak826** [comment]
> as temporary workaround for non-techs, you can tell them to use an "incognito window" / "private window" each time, they get the white page. not really a great solution, but a workaround until the issue is fixed.

as @tldev-de mentioned avoid using normal tab and use incognito / private tab. if you have already logged in using the normal tab clear the cookies before using incognito.

**Comment by charlesBochet** [comment]
This has been fixed in 0.24, closing :)
